 * Serving Flask app 'ar_cam'
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:27100
Press CTRL+C to quit
[ERROR:0@91.700] global D:\a\opencv-python\opencv-python\opencv\modules\videoio\src\cap.cpp (166) cv::VideoCapture::open VIDEOIO(CV_IMAGES): raised OpenCV exception:

OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv\modules\videoio\src\cap_images.cpp:253: error: (-5:Bad argument) CAP_IMAGES: can't find starting number (in the name of file): http://172.30.250.143:8081/video.mjpg in function 'cv::icvExtractPattern'


127.0.0.1 - - [31/Jul/2023 13:06:25] "GET /stream/http://172.30.250.143:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:06:37] "GET /stream/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:07:05] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
[ERROR:0@167.887] global D:\a\opencv-python\opencv-python\opencv\modules\videoio\src\cap.cpp (166) cv::VideoCapture::open VIDEOIO(CV_IMAGES): raised OpenCV exception:

OpenCV(4.6.0) D:\a\opencv-python\opencv-python\opencv\modules\videoio\src\cap_images.cpp:253: error: (-5:Bad argument) CAP_IMAGES: can't find starting number (in the name of file): http://172.30.250.143:8081/video.mjpg in function 'cv::icvExtractPattern'


127.0.0.1 - - [31/Jul/2023 13:07:54] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:08:07] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:08:21] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:08:25] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
127.0.0.1 - - [31/Jul/2023 13:09:29] "GET /snapshot/http://172.30.108.104:8081/video.mjpg HTTP/1.1" 200 -
Invalid camera IP.
[tcp @ 0000015e8a103940] Connection to tcp://172.30.250.143:8081 failed: Error number -138 occurred
[tcp @ 0000015e8a122340] Connection to tcp://172.30.250.143:8081 failed: Error number -138 occurred
